vsintegration/src/FSharp.Editor/DocComments/XMLDocumentation.fs

```diff
@@ -392,12 +392,83 @@ module internal XmlDocumentation =
                 paramName
             )
 
+    [<Literal>]
+    let separatorText = "-------------"
+
     let private AddSeparator (collector: ITaggedTextCollector) =
         if not collector.IsEmpty then
             EnsureHardLine collector
-            collector.Add(tagText "-------------")
+            collector.Add(tagText separatorText)
             AppendHardLine collector
 
+    let BuildSingleTipText (documentationProvider: IDocumentationBuilder, dataTipElement: ToolTipElement) =
+        let mainDescription, documentation, typeParameterMap, exceptions, usage =
+            ResizeArray(), ResizeArray(), ResizeArray(), ResizeArray(), ResizeArray()
+
+        let textCollector: ITaggedTextCollector =
+            TextSanitizingCollector(mainDescription.Add, lineLimit = 45)
+
+        let xmlCollector: ITaggedTextCollector =
```

Comment:
These numbers would benefit from extraction to some constants/variables so that it's easier to make them configurable later.

Reply:
Yes, this is just copypasted from the old method below.
Which I'm not sure gets any use now, but I left it in case some ancient tests use it. 
This file deserves attention in separate pr, tbh.

Reply:
Sure, it's fine to keep it as it is, just observations for some potential future work.